packages/docs/src/components/icons/CmsLogo.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { SvgIcon } from '@cmsgov/design-system';
+
+/**
+ * SVG for CMS Logo
+ */
+const CMSLogoIcon = () => {
+  // "#323a45" was what Brent originally sent
+  const fill = 'var(--color-base)';
+
+  return (
+    <SvgIcon title="CMS logo" viewBox="-2 0 88 34" className="c-cms-logo-icon ds-u-font-size--5xl">
```

Comment:
I don't remember if the SvgIcon component always makes them square, but the logo is ending up square, which makes the header really tall. We probably don't need the SvgIcon component and could do something similar to what we do for the HealthCare.gov logo.

Reply:
Looks much better now!